opcua/ua/uatypes.py

```diff
@@ -409,6 +412,21 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(self.__key())
 
+    def is_null(self):
+        ret = True
+        if self.NamespaceIndex != 0:
```

Comment:
where is the definition of a null node? I cannot find it. I cannot understand that a client can request a nodeid without specifying an namespace index. maybe NodeId(0, 3) if also null? that would solve our issue...